README.rst

```diff
@@ -41,6 +41,27 @@ If a Python core developer approved a PR made by anyone and added the "🤖 auto
 it will be automatically merged once all the CI checks pass.
 
 
+Setup Info
+==========
+
+.. highlight:: shell
+
+Requires Python 3.6+
+
+Create virtual environment and activate it::
+
+    $ python3 -m venv venv
+    $ source venv/bin/activate
```

Comment:
This assumes a unix environment and won't work for Windows I think. Is that an issue?